src/coreclr/src/vm/jitinterface.cpp

```diff
@@ -13351,6 +13385,7 @@ BOOL LoadDynamicInfoEntry(Module *currentModule,
             }
 
         MethodEntry:
+#if defined(TARGET_X86) && defined(TARGET_WINDOWS)
             if (kind == ENCODE_METHOD_NATIVE_ENTRY)
```

Comment:
`ENCODE_METHOD_NATIVE_ENTRY` handling does look quite right:
- It is left over from fragile crossgen, not wired into R2R correctly (e.g. it is missing in inc/readytorun.h)
- It is not handled in R2RDump, crossgen2, etc.

I think it may be easiest to fail crossgen for anything related to native callable on Windows x86 for now (both AOTing the method itself, and also AOTing methods that load the address of native callable method), and open issue on it.

Reply:
(and also delete `ENCODE_METHOD_NATIVE_ENTRY`)

Reply:
Oh. I misinterpreted its meaning. Will cleanup.